### ISSUE TITLE
Leaderboard page not redirecting to home

### ISSUE THREAD

**Issue by Crimson-Typhoon-147** [Vinay-Khanagavi/Cyberbear.github.io#95]
When clicking on the home button/link from the leaderboard page, the redirection does not occur as expected. Instead, it directs to a error 404 page without navigating back to the homepage. This issue affects user experience, as users are unable to return to the main page directly from the leaderboard.

**Comment by B1N4RY-P4R45173** [comment]
You can start with pr, U just have to edit the href ig?

**Comment by Crimson-Typhoon-147** [comment]
Yes @B1N4RY-P4R45173  and one more thing I would Like to add is creating a resume button in the leaderboard page which will redirect the page to the Level the person has left from. 

and can ou also add gssoc label?

**Comment by B1N4RY-P4R45173** [comment]
Sure do it

**Comment by Crimson-Typhoon-147** [comment]
Shall I open another issue and make the pr separately?

**Comment by Crimson-Typhoon-147** [comment]
Hey @B1N4RY-P4R45173 ! I have made a PR fixing the redirection issue . Please review it . 

https://github.com/Vinay-Khanagavi/Cyberbear.github.io/pull/97

**Comment by Crimson-Typhoon-147** [comment]
@B1N4RY-P4R45173 Shall i start working on this feature?

**Comment by Balamurugan-cyber** [comment]
please assign this issue
